build-settings/conventions/src/main/kotlin/convention-plugins.settings.gradle.kts

```diff
@@ -0,0 +1,10 @@
+pluginManagement {
+    plugins {
+
+    }
+    repositories {
+        google()
```

Comment:
we don't need google and gradle portal repositories for most of the plugins

Reply:
somehow, due to magic of settings plugins, we need it these ones in application settings.gradle.kts